.github/workflows/mobile-tsan.yml

```diff
@@ -47,4 +47,4 @@ jobs:
         ./bazelw test \
             --test_env=ENVOY_IP_TEST_VERSIONS=v4only \
             --config=mobile-remote-ci-linux-tsan \
-            //test/common/...
+            //test/common/... //test/cc/...
```

Comment:
nit: adding \ can help to make the code easier to read, e.g.
```
//test/common/... \
//test/cc/...
```

Reply:
Whoops! Sorry, I missed your comment. Will fix in #31099